tests/plugins/toolsets/test_confluence_converter.py

```
@@ -0,0 +1,409 @@
+        html = """
+        <ac:structured-macro ac:name="warning" ac:schema-version="1">
+        <ac:parameter ac:name="title">Alert</ac:parameter>
+        <ac:rich-text-body>
+        <p>Line one.</p>
+        <p>Line two.</p>
+        </ac:rich-text-body>
+        </ac:structured-macro>
+        """
+        result = confluence_storage_to_markdown(html)
+        # All content lines should be blockquoted
+        lines = [l for l in result.split("\n") if l.strip() and "WARNING" in l or "Line" in l]
```

Ruff (0.15.1)

[error] Ambiguous variable name: `l`

(E741)